src/main/java/com/connexta/store/clients/IndexClient.java

```diff
@@ -33,15 +33,15 @@
   @NotBlank private final String indexApiVersion;
 
   public void index(
-      @NotNull final InputStream cstInputStream,
+      @NotNull final InputStream irmInputStream,
```

Comment:
❓ Why change from CST to IRM? Don't we index on CST?

Reply:
We are going to start indexing on IRM instead of CST.

Reply:
Temporary or for keeps?

Reply:
Are you suggesting that this variable name be more generic?

Reply:
Oh, we are indexing with IRM, but we index only on CST terms right?

Reply:
Correct

Reply:
I think it would be fine as a generic `inputStream`

Reply:
I prefer keeping it hardcoded. I think we'll end up having a dynamic set of `data`s in the future where each `data` has a type. As we iterate and support more use cases, I think we can update the type of this parameter, as opposed to the name of this parameter.